src/jssocials.js

```diff
@@ -74,8 +74,8 @@
         },
 
         _initDefaults: function() {
-            this.url = window.location.href;
-            this.text = $.trim($("meta[name=description]").attr("content") || $("title").text());
```

Comment:
Please also keep in mind that twitter has a limit of 140 characters and that the description is actually a description which consists likely out of more then 140 characters.